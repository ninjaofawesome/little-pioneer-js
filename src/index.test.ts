describe('index page', () => {
    test('hello world title to be present', () => {
        const page =  document.body.innerHTML = `<h1 id="title">Hello World</h1>`;
        expect(page).toMatchSnapshot();
      });
});

