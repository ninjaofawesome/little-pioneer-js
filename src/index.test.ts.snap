// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index page hello world title to be present 1`] = `"<h1 id="title">Hello World</h1>"`;
